MapleServer2/Types/Player.cs

```diff
@@ -28,9 +28,8 @@ public class Player
         public byte Gender { get; set; }
 
         // Job Group, according to jobgroupname.xml
-        public bool Awakened { get; set; }
         public Job Job { get; set; }
-        public JobCode JobCode => (JobCode) ((int) Job * 10 + (Awakened ? 1 : 0));
+        public JobCode JobCode { get; set; }
```

Comment:
hell yea

Reply:
zin :( they didn't like it :sadge:
we back to formula :D